frontend/workflows/envoy/src/remote-triage/index.tsx

```diff
@@ -101,6 +110,12 @@ const TriageDetails: React.FC<WizardChild> = () => {
           { name: "Service Cluster", value: metadata?.serviceCluster },
         ]}
       />
+      <Button
```

Comment:
Really minor but it would be cool to make this a smaller icon on the top right similar to the internal flame graph